__tests__/app.test.js

```diff
@@ -39,3 +38,45 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/article_id", () => {
+  test("200: should respond with and article object with correct keys and values", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
```

Comment:
You need to specify the object you expect to come back. 

Because you are expecting a specific object, you can put in the values from the data. 
At the moment, this doesnt check you are getting back the correct object, it just checks your getting A object.